src/ensembl/src/header/Header.scss

```diff
@@ -61,25 +61,14 @@
   }
 }
 
-.aboutEnsembl {
+.topRight {
   display: flex;
-  align-items: baseline;
-  margin-top: 4px;
-  font-size: 12px;
-  color: $light-blue;
-}
-
-.logotypeAbout {
-  height: 10px;
-  fill: $light-blue;
-  margin-left: 0.5rem;
-}
-
-// For disabling temporarily the about Ensembl link in production
-.aboutEnsemblDisabled {
-  color: $medium-dark-grey;
+  align-items: center;
+  margin-left: 20px;
 
-  .logotypeAbout {
-    fill: $medium-dark-grey;
+  .topRightText {
```

Comment:
Confused. What's the purpose for this span?

![image](https://user-images.githubusercontent.com/6834224/121398854-ba49de00-c94d-11eb-9838-45108df62561.png)

Also, what's the purpose of `display: flex` on its immediate parent?

Reply:
sorry that was just copied from the previous launchbar.scss. Makes sense what you say, i will remove it unless the person who originally implemented it had a different expectation but looking at how the homepage is going to be now, seems like its not necessary the extra span.